module5/intermission/index.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Intermission Work
+---
+
+Before starting Module 5, spend a minimum of 2 hours working through these [PostgreSQL Exercises](https://pgexercises.com/).
+
+You **must** use [this tracker](https://docs.google.com/spreadsheets/d/1uCOkwhQEDyPlRIDF2Z9Ff7DvU28aOSX_k0XIML9mnPI/) to keep track of your progress.  Make a copy of the tracker, and rename it so that the title includes your name.
```

Comment:
One more idea! Might be useful here to include a link to the SQL lessons we did in Mod 2 to encourage students to review those as they are ramping back up on SQL.

Reply:
Yea... I was actually debating on linking some resources here - like our lessons and W3Schools.  It would be helpful to students in completing these exercises, but I'm not sure it will be helpful in the long run - they should be at the point where they can handle their own research ?